QueryBuilder/FilteredQuery.cs

```diff
@@ -145,6 +145,20 @@ public TQuery WhereNotIn<TModel>(string propertyName, string[] values, string al
             return (TQuery)this;
         }
 
+        /// <summary>
+        /// Add a where isOfModel operator.
+        /// </summary>
+        /// <param name="alias">Optional - Model Alias.</param>
+        /// <returns>ADT query instance.</returns>
+        public TQuery WhereIsOfModel<TFromModel, TWithModel>(string alias = null)
```

Comment:
Please fix the names for the type arguments and add documentation for them

Reply:
Not sure what names will make sense.